frontend/src/lang/modules/zh.ts

```diff
@@ -369,7 +369,7 @@ const message = {
 
         core: '物理核心',
         logicCore: '逻辑核心',
-        loadAverage: '最近 {0} 分钟平均负载',
+        loadAverage: '最近 1 分钟平均负载 | 最近 {n} 分钟平均负载',
         load: '负载',
         mount: '挂载点',
         fileSystem: '文件系统',
```

Comment:
The provided code difference seems to be related to an update to a JavaScript object that likely represents system monitoring data or configuration. The change you've made involves updating the `loadAverage` property to include both "最近 1 分钟平均负载" (one minute average workload) and "最近 {n} 分钟平均负载" ({n} minutes' average workload). Here's a summary of the changes:

### Changes

```diff
-        loadAverage: '最近 {0} 分钟平均负载',
+        loadAverage: '最近 1 分钟平均负载 | 最近 {n} 分钟平均负载',
```

### Explanation

- **Removed Previous Format**: The old format used `{0}` which was replaced with "最近 1 分钟平均负载".
  
- **Added New Format**: A new option `"最近 {n} 分钟平均负载"` is added, allowing users to specify how many minutes they want to consider for the average load.

### Potential Issues/Critical Points

1. **Dynamic Loading**:
   - Ensure that there are dynamic values (`{n}` in this case) being injected into the string at runtime. If used incorrectly, it could lead to misinterpretation or unexpected output if not properly handled.

2. **UI/UX Considerations**:
   - Be mindful of user interface implications due to the addition of two options for `loadAverage`. It might require additional UI components to handle both formats effectively unless they can be collapsed dynamically or selected by the user.
   
3. **Consistency within Document**:
   - If other parts of the document use different formats for `loadAverage`, maintain consistency across the documentation for clarity.

4. **Documentation Update**:
   - Make sure any relevant documentation around the usage of `loadAverage` reflects these changes, especially mentioning whether both formats support the same parameters if needed.

5. **Testing**:
   - Thoroughly test the updated feature with various scenarios to ensure correct behavior across different inputs and configurations.

If none of these considerations apply to your specific context, your changes look straightforward enough.